docs/configuration.md

```diff
@@ -25,13 +25,17 @@ from flask_pyoidc.provider_configuration import ProviderConfiguration, ProviderM
 
 provider_metadata = ProviderMetadata(issuer='https://op.example.com',
                                      authorization_endpoint='https://op.example.com/auth',
-                                     jwks_uri='https://op.example.com/jwks')
+                                     jwks_uri='https://op.example.com/jwks',
+                                     userinfo_endpoint='https://op.example.com/userinfo')
 config = ProviderConfiguration(provider_metadata=provider_metadata, [client configuration])
 ```
 
 See the OpenID Connect specification for more information about the
 [provider metadata](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderMetadata).
 
+As mentionned in OpenID specification, `userinfo_endpoint` is optionnal. If it's
+not provided, no userinfo request will be done. `flask_pyoidc.UserSession.userinfo` will be set to `None`.  
```

Comment:
Some small typos:
* "As mention~n~ed in OpenID _Connect_ specification"
* "optionnal" -> "optional"

Suggestion:
* "done. `flask_pyoidc.UserSession.userinfo` [...]" -> "done and`flask_pyoidc.UserSession.userinfo` [...]"